app.rb

```diff
@@ -0,0 +1,108 @@
+require_relative 'book'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+
+class App
+  attr_accessor :book_lists, :user_lists, :rental_lists
+
+  def initialize
+    @book_lists = []
+    @user_lists = []
+    @rental_lists = []
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+    case person_type
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid option'
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    puts 'Has parent permission? [y/n]'
+    parent_permission = gets.chomp
+    parent_permission = case parent_permission.downcase
+                        when 'y'
+                          true
+                        else
```

Comment:
- [ ] Please esnure that in your README file you state clearly what step any user (both technical and non-technical users) should take to **_run_** this project. Currently, you have stated this steps clear. See below your current README for this section 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/223960333-7eaeaf74-0122-4083-9c46-f54496076a8f.png)

It can get better. You should mention the entry point of the app i.e `ruby main.rb` .